### ISSUE TITLE
Add the iCub on a seesaw and iCub standup worlds with iCubGazeboV2_5

### ISSUE THREAD

**Issue by gabrielenava** [robotology-playground/icub-gazebo-wholebody#22]
Thanks to the [ConfigurationOverride](https://github.com/robotology/gazebo-yarp-plugins/pull/401) plugin is now possible to modify the robot joint configuration without the need of duplicating the model in case different configurations are required for different demos. It is therefore possible to create a `Gazebo world file` for [icub balancing on a seesaw](https://github.com/robotology/icub-gazebo-wholebody/tree/master/worlds/icub_seesaw_world) and [icub sitting on a chair](https://github.com/robotology/icub-gazebo-wholebody/tree/master/worlds/icub_standup_world) using the [iCubGazeboV2_5](https://github.com/robotology/icub-models/tree/master/iCub/robots/iCubGazeboV2_5) model, without the need of duplicating it (which is the reason why this files have not been created until now). 

I will create these two models. @Yeshasvitvs maybe we can discuss about the `icub-standup` model.

**Comment by yeshasvitirupachuri** [comment]
As discussed f2f I will create a new [gazebo world](https://github.com/loc2/element_reactive-phri/issues/45) with two icubs and a board, the icub models will be ` iCubGazeboV2_5` instead of the old model. As this world will be used with a new controller that will be developed it is better to proceed with the new ` iCubGazeboV2_5` model directly

**Comment by gabrielenava** [comment]
I tried to update the `standup world` to have also a dedicate world for `iCubGazeboV2_5`. Gazebo didn't load the world file I've created, and complained about a missing `model.config` file for the `iCubGazeboV2_5` model. Actually, looking at the models in [icub models](https://github.com/robotology/icub-models/tree/master/iCub/robots/iCubGazeboV2_5) it seems they don't have an associated `model.config`. At the moment, this problem is blocking this issue.

**Comment by traversaro** [comment]
> Actually, looking at the models in icub models it seems they don't have an associated model.config. At the moment, this problem is blocking this issue.

The `model.config` is automatically generate as part of the build process: https://github.com/robotology/icub-models/blob/master/CMakeLists.txt#L40 . 
Are you testing this as part of the robotology-superbuild or with the `icub-models` repo installed manually?

**Comment by gabrielenava** [comment]
>Are you testing this as part of the robotology-superbuild or with the icub-models repo installed manually?

as part of the robotology-superbuild. Ok, so it seems to be a problem of paths. I'll check if it is due to my local configuration or it is more general.

**Comment by gabrielenava** [comment]
I've fixed the issue, it was a local problem caused by my monthly `erase-all-repos-and-reinstall-everything`.

I modified the `seesaw` and `standup` worlds in the `devel` branch to point the `iCubGazeboV2_5` model, as in the `devel` branch the models they were pointing before do not exist anymore. Commit is here: https://github.com/robotology/icub-gazebo-wholebody/commit/1cbc0e3929cb411b5e7b3fb54f6d6bb29cf0c94c

Closing the issue.
